charts/multi-casskop/templates/deployment.yaml

```diff
@@ -50,20 +50,6 @@ spec:
 {{- range .Values.k8s.remote }}
           - -r{{ . }}
 {{- end }}
-        readinessProbe:
```

Comment:
Can you elaborate on this ? This is managed by k8s itself so why remove this ?

Reply:
These are health checks provided with controller-runtime - in casskop we have https://github.com/cscetbon/casskop/blob/master/main.go#L205
But multicluster-controller does not have such functionality.